### ISSUE TITLE
The "file" field seems not supported 

### ISSUE THREAD

**Issue by stefanobasta** [rendezz/dokuwiki-pagemod#31]
It seems that the bureaucracy "file" field isn't supported. The uploaded media can't be stored.

**Comment by ruud12** [comment]
Agree. It used to work for me, but not anymore. No obvious errors are shown unfortunately

**Comment by ruud12** [comment]
Added the processUploads function from the bureaucracy actiontemplate file to the pagemod.php file. It seems that $this>processUploads(fields) is not called. After adding the function the upload works fine again. Pull request has been issued with this fix..

**Comment by Floffyko** [comment]
Thanks for the solution. 
I was using a not very nice way of uploading via bureaucracy action template in pagemod form...

Your solution from here work nice:
[https://github.com/BaselineIT/dokuwiki-pagemod/blob/891b6465a4b2a1db1d548611cc2584b49ec815c1/helper/pagemod.php](url)

`$ns    = $field->getParam('namespace'); // commenting this will break the placeholders`

**Comment by Floffyko** [comment]
Solved the upload problem in simpmle forms, but it breaks the following placeholders:
`action pagemod "casa:locuinte:@@referitoare la| @@@@Plantații Pomi| @@@@Plantații Viță-de-vie| @@@@Plantații Arbuști| @@:horticultura:@@Consemnăm@@" add_puiet`

Can you try to solve this please? Thanks

**Comment by Floffyko** [comment]
I am able to upload the files only on the namespace belonging to the form file. Is there a way, like in bureaucracy, to set the upload path, directly or via some placeholders?
